test/cmdlineTests.sh

```diff
@@ -41,31 +43,33 @@ test "${output//[[:blank:]]/}" = "3"
 
 function compileFull()
 {
-    files="$*"
+    local files="$*"
+
     set +e
-    "$SOLC" --optimize \
-    --combined-json abi,asm,ast,bin,bin-runtime,clone-bin,compact-format,devdoc,hashes,interface,metadata,opcodes,srcmap,srcmap-runtime,userdoc \
-    $files >/dev/null 2>&1
-    failed=$?
+    local output=$( ("$SOLC" $FULLARGS $files) 2>&1 )
+    local failed=$?
     set -e
+
     if [ $failed -ne 0 ]
     then
         echo "Compilation failed on:"
-        cat $files
```

Comment:
I think we should print both the source and the compiler output, otherwise it might be hard to find the input in order to  reproduce.

Reply:
What is the next action here?

Reply:
I can output the command that failed if that helps with debugging, but what I don't find useful is to output the whole contents of the input files concatenated together.

Reply:
@chriseth is above satisfactory?

Reply:
yep, sounds good